terraform/container.go

```diff
@@ -291,6 +288,53 @@ func (terraformContainer *Container) listWorkspaces(errorStream io.Writer) (map[
 	return result, nil
 }
 
+func (terraformContainer *Container) CopyTerraformLockIfExists(outputStream, errorStream io.Writer) error {
+	lockExists, err := terraformContainer.CheckFileExists("/build/.terraform.lock.hcl", errorStream)
+	if err != nil {
+		return err
+	}
+
+	if lockExists {
```

Comment:
Suggest simplifying this by handling the case where the lockfile doesn't exist first and doing an early return (means you don't need to indent the other case in an else).